site/_data/master-toc.yml

```diff
@@ -4,27 +4,33 @@ toc:
       - page: About Velero
         url: /index.html
       - page: How Velero works
-        url: /about
+        url: /how-velero-works
       - page: About locations
         url: /locations
-      - page: Supported platforms
-        url: /support-matrix
   - title: Install
     subfolderitems:
       - page: Overview
         url: /install-overview
       - page: Upgrade to 1.1
         url: /upgrade-to-1.1
-      - page: Quick start with in-cluster MinIO
```

Comment:
The idea with the quick start was to have a place for new users to quickly get up and running for evaluation/experimentation - by removing it from the TOC, that guide becomes buried. Personally I'd opt to keep it around in the TOC - what do y'all think?

Reply:
I guess I resist this because that documentation is not for production, and it would be the only thing not production worthy in this entire section. Could we add it as a `Quick start` submenu to either the `Use` or `Contributing` section?

Reply:
Personally I'd rather have it higher up since the point of it is to be a good starting point for new users. Let's see what @nrb thinks. Also FYI Istio has something similar in their Install section (I think their docs are pretty reasonably organized): https://istio.io/docs/setup/install/

Reply:
Ok, I like `evaluation install`, that fits.

We can also move the Helm installation instructions up to a subitem if we can add more content to it. Not wanting to do that now tho.

Reply:
In my own reading, a "Quick start" isn't meant to be production ready. But I like the "Evaluation install" title and some wording inside the page itself.